ember-toucan-core/src/components/form/textarea-field.ts

```diff
@@ -1,15 +1,16 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
+import type { ErrorMessage } from '../../-private/types';
```

Comment:
Are we exporting / going to export types via an `index` file, or do these types get exported already?

Just thinking this would be useful for folks using the components...